runtime-modules/proposals/engine/src/tests/mod.rs

```diff
@@ -2714,123 +2708,3 @@ fn proposal_status_resolution_is_slashing_achievable_works_correctly() {
     assert!(slashing_threshold_achievable_resolution.is_slashing_threshold_achievable());
     assert!(!slashing_threshold_achievable_resolution.is_rejection_imminent());
 }
-
-#[test]
-fn proposal_status_resolution_slashing_threshold_works_correctly_with_abstentions() {
```

Comment:
Maybe it's worth leaving these test since they provide explicit evidence that abstentions are effectively accounted for. WDYT?

Reply:
Well those tests were were passing a value for `abstentions` field in `ProposalStatusResolution`. This field was dropped. So it doesn't make sense to  add those tests again.

Reply:
But it could make sense to add some more tests of this type https://github.com/Joystream/joystream/blob/carthage/runtime-modules/proposals/engine/src/tests/mod.rs#L532 to confirm Abtention is equivalent to reject?

Reply:
What I was suggesting was:
Have a test with `abstentions > 0` and having the correct quorum ratio